.circleci/config.yml

```diff
@@ -43,6 +43,17 @@ jobs:
                       - node_modules
                   key: v1-dependencies-{{ checksum "yarn.lock" }}
             - run: yarn build
+            - run: ls
             - run: yarn lint
             - run: yarn prettier:ci
+            - setup_remote_docker # Expose the docker daemon so that it can be used in tests.
+            - run: yarn test
+            - run: docker ps
+            - run: ls
```

Comment:
what's `docker ps` and `ls` for? Just for debugging?

Reply:
Yep! I will remove once I figure out the underlying problem